ext/web/06_streams.js

```diff
@@ -1359,7 +1350,7 @@ function readableByteStreamControllerEnqueue(controller, chunk) {
       "chunk's buffer is detached and so cannot be enqueued",
     );
   }
-  const transferredBuffer = transferArrayBuffer(buffer);
+  const transferredBuffer = ArrayBufferPrototypeTransfer(buffer);
```

Comment:
Oh, `[[ArrayBufferMaxByteLength]]` internal slot is not carried over, so `ArrayBufferPrototypeTransferToFixedLength` should be used.
https://streams.spec.whatwg.org/#transfer-array-buffer